config/base/numaflow-server/numaflow-server-metrics-proxy-config.yaml

```diff
@@ -30,28 +30,30 @@ data:
             - namespace
             - mvtx_name
           dimensions:
+            - name: mono-vertex
+              # expr: optional expression for prometheus query
+              # overrides the default expression
             - name: pod
               # expr: optional expression for prometheus query
               # overrides the default expression
               filters:
                 - name: pod
                   required: false
+        - metric_name: monovtx_sink_time_bucket
```

Comment:
@whynowy @vigith is this metric name changed recently as we do not see the any value for this metric in prometheus locally.

Reply:
i did not find the setting/implementation of this [metric](https://github.com/numaproj/numaflow/blob/1c0989a9667b3c664bc965655c40bc05d4537384/rust/numaflow-core/src/metrics.rs#L83). It might have been removed in some PR? Because in earlier versions, we can see the data coming.

Reply:
on port-forwarding the metrics for monovertex - some of them are missing
<img width="889" alt="image" src="https://github.com/user-attachments/assets/5e0f37b9-a06f-412b-91c6-8d4173899dbf">

Reply:
this is bug in the current main when streaming was introduced. @yhl25 let's fix this in a follow-up